lib/driver.js

```diff
@@ -182,15 +185,38 @@ class YouiEngineDriver extends BaseDriver {
     let iosArgs = {
       javascriptEnabled: true,
     };
-    let iosdriver = new IOSDriver(iosArgs);
+    let driverName = "IOSDriver";
+    if (opts.platformVersion) {
+      let majorVer = opts.platformVersion.toString().split(".")[0];
```

Comment:
So specifying the platformVersion is mandatory now? Must be sure to update any documentation about that.